apps/shipments/migrations/0036_historicallocation_historicalshipment.py

```diff
@@ -0,0 +1,141 @@
+                ('delivery_est', models.DateTimeField(blank=True, null=True)),
+                ('delivery_act', models.DateTimeField(blank=True, null=True)),
+                ('delivery_attempt', models.DateTimeField(blank=True, null=True)),
+                ('cancel_requested_date_act', models.DateTimeField(blank=True, null=True)),
+                ('cancel_confirmed_date_act', models.DateTimeField(blank=True, null=True)),
+                ('customs_filed_date_act', models.DateTimeField(blank=True, null=True)),
+                ('customs_hold_date_act', models.DateTimeField(blank=True, null=True)),
+                ('customs_release_date_act', models.DateTimeField(blank=True, null=True)),
+                ('container_type', models.CharField(blank=True, max_length=255, null=True)),
+                ('port_arrival_locode', models.CharField(blank=True, max_length=255, null=True)),
+                ('final_port_locode', models.CharField(blank=True, max_length=255, null=True)),
+                ('import_locode', models.CharField(blank=True, max_length=255, null=True)),
+                ('lading_locode', models.CharField(blank=True, max_length=255, null=True)),
+                ('origin_locode', models.CharField(blank=True, max_length=255, null=True)),
+                ('us_routed', models.NullBooleanField()),
+                ('import_customs_mode', models.CharField(blank=True, max_length=255, null=True)),
+                ('us_export_port', models.CharField(blank=True, max_length=255, null=True)),
+                ('version', models.CharField(default='1.2.1', max_length=255)),
+                ('trailer_number', models.CharField(blank=True, max_length=255, null=True)),
+                ('seal_number', models.CharField(blank=True, max_length=255, null=True)),
+                ('is_master_bol', models.NullBooleanField()),
+                ('nmfc_class', models.CharField(blank=True, max_length=4, null=True, validators=[django.core.validators.RegexValidator(message='Invalid nmfc class number.', regex='^1[015]0$|^1[27]5$|^2[05]0$|^[345]00$|^[567]0$|^[568]5$|^92\\.5$|^77\\.5$')])),
+                ('is_hazmat', models.NullBooleanField()),
+                ('customer_fields', django.contrib.postgres.fields.jsonb.JSONField(blank=True, null=True)),
+                ('history_id', models.UUIDField(default=uuid.uuid4, editable=False, primary_key=True, serialize=False)),
+                ('history_date', models.DateTimeField()),
+                ('history_change_reason', models.CharField(max_length=100, null=True)),
+                ('history_type', models.CharField(choices=[('+', 'Created'), ('~', 'Changed'), ('-', 'Deleted')], max_length=1)),
+                ('history_user', models.CharField(blank=True, max_length=36, null=True)),
+                ('bill_to_location', models.ForeignKey(blank=True, db_constraint=False, null=True, on_delete=django.db.models.deletion.DO_NOTHING, related_name='+', to='shipments.HistoricalLocation')),
```

Comment:
After much thought, I think we do not want our location fields on `HistoricalShipment` to reference `HistoricalLocation`. The point of the `HistoricalShipment` model is to track the state of the `Shipment` model, so `bill_to_location`, for example, should always point to a valid `Location` ID, not a `HistoricalLocation` ID. This should also allow us to simplify a good bit of code.